source/_patterns/01-molecules/content-header/content-header.yaml

```diff
@@ -1,2 +1,10 @@
 contentTitle:
   text: Content title
+subjects:
```

Comment:
Wondering whether we need a different term.

Reply:
Agree `subjects` is very eLife-specific. Could imagine other categorisations may apply from other publishers. Opinions on `categories`?

Reply:
I will switch to categories for now and then we can discuss.

Reply:
I have switched it to `categories` for now.